test/hotspot/jtreg/vmTestbase/nsk/jvmti/scenarios/hotswap/HS201/hs201t002/hs201t002.cpp

```diff
@@ -99,17 +95,19 @@ void disableEvent(jvmtiEnv *jvmti_env, jvmtiEvent event, jthread thread) {
 void redefineClass(jvmtiEnv *jvmti_env, jclass klass) {
 
     jvmtiClassDefinition classDef;
-
     char *className;
+    jint newClassSize;
+    unsigned char* newClassBytes;
 
     if (!NSK_JVMTI_VERIFY(jvmti_env->GetClassSignature(klass, &className, NULL))) {
         nsk_jvmti_setFailStatus();
         return;
     }
 
-    if (!NSK_VERIFY(readNewBytecode(jvmti_env))) {
+    if (!NSK_VERIFY(readNewBytecode(jvmti_env, newClassSize, newClassBytes))) {
```

Comment:
For simple types I don't care for C++ pass-by-ref arguments. At the call site it just looks like the values are being passed in. When I first read this it didn't look right to me since they are uninitialized, so I had to look at readNewBytecode() to see what was going on.

Reply:
Changed arguments to pointers